### ISSUE TITLE
Computer Vision Notebook 3 

### ISSUE THREAD

**Issue by Hrithik2212** [mrdbourke/pytorch-deep-learning#88]
I tried the CNN model with Adam optimizer with the same code , however the validation accuracy didn't increase and stayed a 10 % in all the epochs , I am not sure why this happens

**Comment by mrdbourke** [comment]
Hi @Hrithik2212,

Could you be more specific?

Which section of the notebook are you talking about?

You can see the notebook here: https://www.learnpytorch.io/03_pytorch_computer_vision/

Did you try stepping through each line of code and seeing what happens?

**Comment by Hrithik2212** [comment]
I am sorry to disturb you , I just deleted and redid the code and it worked 

I got 89% accuracy in my first epoch on validation set and didn't improve in subsequent epochs 
Any suggestions to improve accuracy
